src/Parser/BloodDeathKnight/Modules/Items/T20_2pc.js

```diff
@@ -11,22 +11,37 @@ import Combatants from 'Parser/Core/Modules/Combatants';
 class T20_2pc extends Module {
   static dependencies = {
     combatants: Combatants,
-  }
+  };
 
+  uptime=0;
 
   on_initialized() {
     this.active = this.combatants.selected.hasBuff(SPELLS.BLOOD_DEATH_KNIGHT_T20_2SET_BONUS_BUFF.id);
   }
 
+  on_finished() {
+    this.uptime = this.owner.modules.combatants.getBuffUptime(SPELLS.GRAVEWARDEN.id) / (this.owner.fightDuration);
```

Comment:
Only updating the value in `on_finished` is undesirable as it won't show progress while loading.